packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -0,0 +1,316 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+
+import 'package:google_maps_flutter_platform_interface/src/method_channel/method_channel_google_maps_flutter.dart';
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+/// The interface that implementations of google_maps_flutter must implement.
+///
+/// Platform implementations should extend this class rather than implement it as `google_maps_flutter`
```

Comment:
Yes, this phrase is broken, it seems there's a few lines missing. Let me fix this with the whole thing (it's formulaic, and is repeated in other plugins as well)

Reply:
It should be something similar to [this](https://github.com/flutter/plugins/blob/master/packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart#L14-L20).

Thanks for noticing!

Reply:
(It is correct. It means that you should never `implements GoogleMapsFlutterPlatform` :P)

Reply:
I've rewritten this as:

```
/// The interface that platform-specific implementations of `google_maps_flutter` must extend.
///
/// Avoid `implements` of this interface. Using `implements` makes adding any new
/// methods here a breaking change for end users of your platform!
/// 
/// Do `extends GoogleMapsFlutterPlatform` instead, so new methods added here are
/// inherited in your code with the default implementation (that throws at runtime),
/// rather than breaking your users at compile time.
```